java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -1636,4 +1639,29 @@ public void zcard_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(value, payload);
     }
+
+    @SneakyThrows
+    @Test
+    public void zcount_returns_success() {
```

Comment:
we should also have a test with a ScoreBoundary set to false.

Reply:
These are tested in IT tests extensively.